queue_services/entity-emailer/src/entity_emailer/worker.py

```diff
@@ -142,6 +143,9 @@ def process_email(email_msg: dict, flask_app: Flask):  # pylint: disable=too-man
         elif etype and etype == 'bc.registry.bnmove':
             email = bn_notification.process_bn_move(email_msg, token)
             send_email(email, token)
+        elif etype and etype == 'bc.registry.dissolution':
```

Comment:
can we add a check to confirm the `data.furnishingName` is one of the following:

- DISSOLUTION_COMMENCEMENT_NO_AR
- DISSOLUTION_COMMENCEMENT_NO_TR
- DISSOLUTION_COMMENCEMENT_NO_AR_XPRO
- DISSOLUTION_COMMENCEMENT_NO_TR_XPRO

Reply:
Updated